### DISCUSSION TITLE
[RFC] RangePicker label modify

### DISCUSSION THREAD

**Discussion by zombieJ** [ant-design/ant-design#45870] [RFCs]
# 📋 RFC

## Summary

RangePicker support customize label for display.

## Motivation

ref #32864

https://user-images.githubusercontent.com/50199114/143213320-574a6599-b22e-4051-8aef-dca8bfa56ac1.mov

## API design

```tsx
const [focused, setFocused] = React.useState(false);

const myFormat = (date, info: { focused, range: 'start' | 'end' }) => focused ? labelStateValue : 'YYYY-MM-DD';

<RangePicker
  format={[
    myFormat,
    // Multiple like: 'YYYY-MM-DD',
  ]}
/>
```

**Comment by afc163** [comment]
inputRender

**Reply by zombieJ** [reply]
`inputRender` 比较有歧义，看起来像是输入框的 render，但是这个 label replace 需要是两个一起。

**Comment by crazyair** [comment]
https://ant-design.antgroup.com/components/date-picker-cn#components-date-picker-demo-format

`format={(values, render) => render([xxx, yyy])}`

`render` contain `separator`

**Comment by selicens** [comment]
api style 1

**Comment by crazyair** [comment]
style 1 should be. ref: https://ant-design.antgroup.com/components/select-cn#select-demo-label-in-value

```tsx
const [value, setValue] = React.useState<{ label: string, value: string }>(null);

<RangePicker
  labelInValue
  value={value}
  onChange={(value) =>  setValue(value) }
/>

```
